pkg/extension/sumologicextension/extension.go

```diff
@@ -592,6 +618,135 @@ func (se *SumologicExtension) sendHeartbeatWithHTTPClient(ctx context.Context, h
 	return nil
 }
 
+func getHostIpAddress() (string, error) {
+	c, err := net.Dial("udp", "8.8.8.8:53")
```

Comment:
What if this specific DNS server is filtered in customers environment. Do we care about such situations?
Maybe the address should be configurable at some point, and/or we should use some fallback if this method won't work

Reply:
I don't think we actually connect to this address, we just need the connection object to check the local address. I'd change it to an obviously invalid address (`255.255.255.255` maybe?) and add a comment to make this clear.

Reply:
Correct, we don't connect. I liked the idea of using a known/typical DNS host/port, but this causes more confusion 😄 I'll use your suggestion.